bindings/cpp/logger.cpp

```diff
@@ -13,11 +13,17 @@
 #include <blackhole/record.hpp>
 #include <blackhole/logger.hpp>
 
+#include "example/config.hpp"
 #include "elliptics/session.hpp"
 #include "library/elliptics.h"
 
 namespace ioremap { namespace elliptics {
 struct trace {
+	struct trace_stack_t {
```

Comment:
std::stack is dynamic structure. It will allocate and free memory as needed. We don't want any memory allocation in the runtime.